### ISSUE TITLE
"Social Sprints" should let friends interact with a writing sprint

### ISSUE THREAD

**Issue by drusepth** [drusepth/Indent#122]
Ways to interact:
1. You create a Social Sprint and specify how long it should minimally last and share it with Facebook, Google+, Twitter, etc. 
2. When you start, the above posts are shared, and you have the standard writing interface (though could be zen mode) as well as two numbers: a countdown from the time you specified, and a count for your target word count (determined by an initial writing setup or from some metrics). The target word count counts down with each word you write. 
3. Every time someone +1's, reshares, likes, comments, retweets, mentions, etc your post, you see a small notification in the writing window with that person's name, photo, and a short message that says, "I'm +1 behind you! Go, go, go!" or "Andrew says: You can do it!" (in the case of a comment, insert the comment). And based on your writing speed/goal/others, you are also given X more words on your target, and X minutes added.

So basically, you could start a 5 minute sprint (which puts you at, say, 500 words), post it to Facebook, and during that 5 minutes get a ton of likes and comments which continuously cheers you on from friends, and you end up writing for 9 minutes and do 800 words.
